modules/guice/src/test/java/org/mule/module/guice/MuleObjectCreationTestCase.java

```diff
@@ -1,138 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.module.guice;
-
-import org.mule.api.transformer.Transformer;
-import org.mule.api.transport.Connector;
-import org.mule.tck.junit4.AbstractMuleContextTestCase;
-import org.mule.tck.testmodels.fruit.RedApple;
-import org.mule.tck.testmodels.mule.TestConnector;
-import org.mule.transformer.types.DataTypeFactory;
-
-import com.google.inject.AbstractModule;
-import com.google.inject.Provides;
-import com.google.inject.Singleton;
-import com.google.inject.name.Named;
-import com.google.inject.name.Names;
-
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
-public class MuleObjectCreationTestCase extends AbstractMuleContextTestCase
```

Comment:
that tests was verifying prototypes with the GuiceRegistry which is not part of the distribution since a long time ago and which we're actually removing from the code base in 3.7. So, there was no point in refactoring this test to work with the new changes.